setup.py

```diff
@@ -66,7 +66,6 @@ def _get_package_version() -> str:
         "Operating System :: Microsoft :: Windows",
         "Operating System :: MacOS :: MacOS X",
         "Operating System :: POSIX :: Linux",
-        "Programming Language :: Python :: 3.7",
```

Comment:
finally 😄 BTW don't we neet to remove it from [here](https://github.com/dbt-athena/dbt-athena/blob/057aa45b1f40ba0181b5fec35e7b7441da0bd227/.github/workflows/ci.yml#L24) as well ?